data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1010 @@ mission "Unfettered Jump Drive Trading"
+			action
+				"unfettered rep: Si-Ra" --
+			label notagain
+			`	"We needed to act discreetly, so we would have the time to check the status of the jump drives without being swarmed by opportunist pirates. More people would have only attracted more attention and been more targets for the vault's defenses against. As ye have probably noticed, we have not accessed this vault for quite some time. Some unfettered humans could have snuck in and sabotaged the jump drives, like they did with the turrets." You have a hunch that there could be more to all this than what he's admitting.`
+				goto ship
+			label vault
+			`	"Yes and no. It be not only a holographic generator, but also a set of jammers, meaning your ship would not be able to pinpoint the exact location and would have to land blindly. We also needed to make sure the jump drives were not sabotaged and that the vault defenses were not compromised; they could probably have taken yer ship down. Needless to say, even monkeys could rig the drives to blow when used. Checking that takes time." He takes some time to think before continuing, "Besides, we want to be using this vault later, so may as well ensure its safety. That said, it seems we left it sit too long, it be too compromised for further use." You have a hunch that there could be more to all this than what he's admitting.`
+			choice
+				`	(Go back to the ship.)`
+					goto ship
+				`	"If this world is so wild, why put the jump drives here in the first place?"`
+			`	"We be not the ones who placed these drives here..." The Hai doesn't seem to want to tell you much about this vault.`
+			label ship
+			`	As soon as your ship arrives, you give the orders to collect the jump drives and install them if possible, so you may flee more swiftly. When your ship takes off, you notice more activity than on the way in...`
+			`	The Hai comes to speak to you on the bridge. "Batten down the hatches, matey! It took us too long to check and load the drives, and we seem to have attracted attention. I guess a lookout got away from the vault and contacted some friends..." Multiple alarms start going off, confirming his suspicions - hostile ships are launching and heading straight for you. You launch as soon as you can, rushing through the correct procedures. Meanwhile, the Unfettered captain joins you on the bridge, clearly used to being where the action is, invited or not, giving you advice. "They'll probably be launching after us, you should first get some distance from them before trying to jump away."`
+				launch
+	on accept
+		outfit "Jump Drive" 4
+		clear "unfettered: dumb Si-Ra question"
+	npc evade
+		government "Cerberus"
+		personality disables plunders vindictive
+		system "Almaaz"
+		fleet "Small Northern Pirates" 2
+	npc evade
+		government "Cerberus"
+		personality disables plunders vindictive waiting
+		system "Almaaz"
+		fleet "Large Northern Pirates"
```

Comment:
Kind of a small fleet, considering first last has been able to get through the firefight in Wah Ki without problems.

Reply:
if you came with a JD you get extra ships
from experience playing it it wasnt too easy, you cant quite fight them in a hostile system

Reply:
Forgot about the random spawns. You’re right.